<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dual</class>
 <widget class="QMainWindow" name="Dual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/logo.qrc">
    <normaloff>:/icons/48x48/vlc-qt.png</normaloff>:/icons/48x48/vlc-qt.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="group1">
      <property name="title">
       <string>Instance 1</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="open1">
         <property name="text">
          <string>Open 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VlcWidgetVideo" name="video1" native="true"/>
       </item>
       <item>
        <widget class="QPushButton" name="stop1">
         <property name="text">
          <string>Stop 1</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group2">
      <property name="title">
       <string>Instance 2</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="open2">
         <property name="text">
          <string>Open 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VlcWidgetVideo" name="video2" native="true"/>
       </item>
       <item>
        <widget class="QPushButton" name="stop2">
         <property name="text">
          <string>Stop 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header>../../src/widgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>Dual</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
